lib/active_record_florder.rb

```diff
@@ -55,6 +55,14 @@ def step(value)
     def get_step
       @step || 2**16
     end
+
+    def populate(value)
+      @populate = value
+    end
+
+    def get_populate
```

Comment:
I wrote `attr_accessor` as example how it is defining getter and setters. But I see you are using this style in rest of code also. I just wanted point that you probably mix 2 conventions one is eg. `set_populate(value)`, `get_populate()` and the other one is `populate(value)`, `populate()` and you are using one from both. But lets skip this.